code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -166,12 +166,17 @@
 
 /datum/reagent/drug/krokodil/addiction_act_stage4(mob/living/carbon/human/M)
 	CHECK_DNA_AND_SPECIES(M)
-	if(!istype(M.dna.species, /datum/species/krokodil_addict))
-		to_chat(M, "<span class='userdanger'>Your skin falls off easily!</span>")
-		M.adjustBruteLoss(50*REM, 0) // holy shit your skin just FELL THE FUCK OFF
-		M.set_species(/datum/species/krokodil_addict)
+	if(istype(M.dna.species, /datum/species/human) || istype(M.dna.species, /datum/species/human/felinid)) // probably not the best way to do it but until we get a krokodil_addict variant for every race only humans/felinds should be transformed
```

Comment:
This is wrong. `/datum/species/human/felinid`` is `/datum/species/human, so second check does nothing.
also we have `iscatperson()` and `ishumanbasic()` for that

Reply:
also is this even needed? do other species not transform into addicts?

Reply:
they transform into THE krokodil_addict. witch is a human subtype. we dont want nonhuman races to turn into a human...ish thing as that's stupd.

Reply:
All races are human subtypes.

Reply:
thecnically yes but im being more specific as if the zombies have the properites of humans. not the race the player was before they got converted.

Reply:
Still your check doesn't work

Reply:
i mean... i did test it. and it does work so i dunno

Reply:
They just suggested a better version 😆